coalib/results/SourceRange.py

```diff
@@ -126,6 +126,29 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Default representation::
+            filename: FROM start_line:start_column TO end_line:end_column
+        If whole line is affected::
+            filename: FROM start_line TO end_line
+        If whole file is affected::
+            filename
```

Comment:
Yes this is more readable and better to understand :smile: 
`L3 C3 : L10 C7` looks good I guess